### ISSUE TITLE
Optifine Cape New Editor

### ISSUE THREAD

**Issue by MoptopX** [sp614x/optifine#2582]
The banner systems produce capes that in my opinion don't look very good, and none of the capes look good with elytra on either as they use the same texture, my idea was to create some type of online pixel editor for the capes or something along those lines. Good idea?

**Comment by Hax4Ever** [comment]
Fully custom capes are not happening.

**Comment by iPollox** [comment]
People will abuse it, writing bad things or drawing things that kids should be looking at.

**Comment by rayblon** [comment]
> People will abuse it, writing bad things or drawing things that kids should be looking at.

Adding some more shapes to use would be nice, though.

**Comment by MoptopX** [comment]
yeah, we need atleast something that looks better than just banners. Minecon capes have all these cool backgrounds and patterns and shading, like I understand why we cant do a full customizability editor, but still something a little MORE customizable would be nice. Oh yeah also if thats not possible we atleast need to be able to change our elytra textures to something other than the cape ones. It looks really bad with most capes when its like that.

**Comment by Vexima** [comment]
Just be glad it's no longer a colour and the letters OF.

> People will abuse it, writing bad things or drawing things that kids should be looking at.
This is why sp614x is very unlikely to allow you to do this.

**Comment by rayblon** [comment]
> Just be glad it's no longer a colour and the letters OF.

When I was a child, every time I was faced with a challenge I was struggling with, my mother would give me a 'bootstraps' speech, usually a story about the time when she had no air conditioning, had to work two jobs to keep food on the table 6 days of the week, and the most advanced piece of technology she seemed to own was a typewriter. She'd always end it with something along the lines of 'you've got it better now than I ever did, be happy you weren't in my shoes because it was way harder back then'.

It didn't change the fact that the tools I had then still left me struggling. It only served to diminish the perception of the things I *did* accomplish.

So, with all due respect... "just be happy you don't have to deal with X like we did" is not a valid refutation of a suggestion, and it never will be.
